### DISCUSSION TITLE
[Question]: Motion detection activation/deactivation

### DISCUSSION THREAD

**Discussion by AghilasMessara** [blakeblackshear/frigate#13209] [Ask A Question] [answered]
### What is your question:

Hello team,
I can't find in the new UI a way to disable motion detection for a camera. Only the object detection can be toggled. I can't remember now if it was possible in the old UI.
It is still available through MQTT.

**Comment by NickM-27** [comment, marked as answer]
It wasn't in the old UI, in general motion detection disables most things including the live camera switching so it's not in the ui
